airflow-core/docs/security/secrets/secrets-backend/index.rst

```diff
@@ -64,12 +66,21 @@ The ``[secrets]`` section has the following options:
     [secrets]
     backend =
     backend_kwargs =
+    backends_order =
 
 Set ``backend`` to the fully qualified class name of the backend you want to enable.
 
 You can provide ``backend_kwargs`` with json and it will be passed as kwargs to the ``__init__`` method of
 your secrets backend.
 
+``backends_order`` comma-separated list of secret backends. These backends will be used in the order they are specified.
+Please note that the ``environment_variable`` and ``metastore`` are required values and cannot be removed
+from the list. Supported values are:
+
+* ``custom``: Custom secret backend specified in the ``secrets[backend]`` configuration option.
+* ``environment_variable``: Standard environment variable backend ``airflow.secrets.environment_variables.EnvironmentVariablesBackend``.
+* ``metastore``: Standard metastore backend ``airflow.secrets.metastore.MetastoreBackend``.
```

Comment:
I also request you to have some user guidelines in the docs to configure it properly.

For example:

Lets say DB lookup has higher precedence than that of say ENV backend.
This would be shooting ourselves in the foot by compromising the performance here? DB lookup will be more expensive than DB.